core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineMysql.java

```diff
@@ -354,4 +355,16 @@ public String getEscape(LikeExpression likeExpression) {
   public String tryAddIfNotExistsToCreateIndexSql(String createIndexSql) {
     return createIndexSql;
   }
+
+  @Nullable
+  @Override
+  public String rawCatalogName(String namespace) {
+    return namespace;
+  }
+
+  @Nullable
+  @Override
+  public String rawSchemaName(String namespace) {
+    return namespace;
```

Comment:
If the MySQL driver use the catalog name, can we return null here?
```suggestion
    return null;
```

Reply:
Good point. First, I had the same idea. But I found https://dev.mysql.com/doc/connector-j/en/connector-j-connp-props-connection.html:

- databaseTerm

> MySQL uses the term "schema" as a synonym of the term "database," while Connector/J historically takes the JDBC term "catalog" as synonymous to "database". This property sets for Connector/J which of the JDBC terms "catalog" and "schema" is used in an application to refer to a database. The property takes one of the two values "CATALOG" or "SCHEMA" and uses it to determine (1) which Connection methods can be used to set/get the current database (e.g. 'setCatalog()' or 'setSchema()'?), (2) which arguments can be used within the various 'DatabaseMetaData' methods to filter results (e.g. the catalog or 'schemaPattern' argument of 'getColumns()'?), and (3) which fields in the result sets returned by 'DatabaseMetaData' methods contain the database identification information (i.e., the 'TABLE_CAT' or 'TABLE_SCHEM' field in the result set returned by 'getTables()'?). If "databaseTerm=CATALOG", 'schemaPattern' for searches are ignored and calls of schema methods (like 'setSchema()' or get 'Schema()') become no-ops, and vice versa.

It seems which field `catalog` or `schemaPattern` parameter is used can be controlled by the property. I don't think this property is passed by users, but it's technically possible. So, I think it's a slightly safer to return the namespace from `getRawSchemaName()` as well. What do you think? I don't have a strong opinion on it, though.

Reply:
Understood. I don't have a strong opinion on it either, so let's keep it as it is. Thanks.